gaia/formats.py

```diff
@@ -0,0 +1,10 @@
+import itertools
+
+JSON = ['.json', '.geojson']
```

Comment:
@aashish24 a type is actually a source (I could rename it to make it clearer) such as 'file', and format is meant to be more specific (ie a geojson file or a shapefile).